unime/src/i18n/en/index.ts

```diff
@@ -288,7 +288,7 @@ const en = {
     },
   },
   DOMAIN_LINKAGE: {
-    TITLE: 'Verified',
+    TITLE: 'Verified website',
```

Comment:
Is using the term "website" added to make it more relateable for the user? We are not really verifying a website (but the domain), so I think that could be a bit misleading. Shouldn't we just stick with "Verified" and explain more in the dropdown?

Reply:
This was a request by @JelleMillenaar.